en/docs/administer/admin-overview.md

```diff
@@ -0,0 +1,7 @@
+# WSO2 API Manager Administration Overview
+
+This section explains how to carry out the following administration activities for WSO2 API Manager.
+
+- [Applying WUM Updates]({{base_path}}/administer/admin-overview)
```

Comment:
Isn't WSO2 Updates supposed to come in here as well?

Reply:
Moved it to the Setup section